poi-examples/src/main/java/org/apache/poi/examples/xwpf/usermodel/BetterHeaderFooterExample.java

```diff
@@ -48,7 +49,7 @@ public static void main(String[] args) throws IOException {
             XWPFFooter foot = doc.createFooter(HeaderFooterType.DEFAULT);
             foot.createParagraph().createRun().setText("footer");
 
-            try (OutputStream os = new FileOutputStream(new File("header2.docx"))) {
+            try (OutputStream os = Files.newOutputStream(new File("header2.docx").toPath())) {
```

Comment:
Instead of creating `File` object and then using `toPath()` to convert to `Path`, you can directly create `Path` object with `java.nio.file.Paths::get` static method.